boards/FSM/Src/main.c

```diff
@@ -173,19 +175,23 @@ int main(void)
     Io_SharedHardFaultHandler_Init();
 
     Io_FlowMeters_Init(&htim4);
-    primary_flow_meter = App_FlowMeter_Create(Io_FlowMeters_GetPrimaryFlowRate);
-    secondary_flow_meter =
-        App_FlowMeter_Create(Io_FlowMeters_GetSecondaryFlowRate);
+    primary_flow_meter =
```

Comment:
So the min and max values for flow rate and wheel speed should be placed into a header file (`Inc/App/configs/App_FlowRateThresholds.h` and `Inc/App/configs/App_WheelSpeedThresholds`) so that they can be shared between the embedded target and test files. We don't want to hard code values everywhere.

Reply:
Thats a good point. I added the header files.